### PR TITLE
Drizzle monorepo announcement blog post

### PR DESCRIPTION
**Pull request by adrianmcli**
A blog post to announce Drizzle's move to a monorepo. Thanks @cds-amal for all his hard work.

### CODE REVIEW

File: src/blog/drizzle-150-a-new-beginning.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  <figcaption style="text-align: center;">_Easily spin up real front ends to test that dapps can work with the current code_</figcaption>
+</figure>
+
+## Better release strategy
+
+Perhaps the biggest thing that the changes above allow us to do is more frequent releases.
+
+In the past year, we often had to deal with delaying the publishing of a new change because it required testing and updating other repos. Our publishing process was essentially being blocked by having to synchronize changes amongst multiple repos.
+
+With a monorepo that all changes, it means that you can look forward to more regular releases and a faster turnaround time for getting issues fixed. With each release, better changelogs are also coming, as we move towards adopting [conventional commits](https://www.conventionalcommits.org) for each PR that gets merged.
```
Comment (tcoulter) [inline src/blog/drizzle-150-a-new-beginning.md]: This sentence is really hard to parse. 

> With a monorepo that all changes, it means that you can look forward to more regular releases and a faster turnaround time for getting issues fixed.

I got stopped up after the word "monorepo", and needed a comma there to better understand the sentence. Also, the use of "it means" after the current comma doesn't flow well. I'd change to one of the following options (my changes bolded): 

1. With a monorepo, that all changes, **which** means that you can look forward to more regular releases and a faster turnaround time for getting issues fixed.
1. With a monorepo, that all changes, **which** means ~that you can look forward to~ more regular releases and a faster turnaround time for getting issues fixed.
1. With a monorepo, that all changes, **meaning** ~that you can look forward to~ more regular releases and a faster turnaround time for getting issues fixed

Something like that. :) 

Also, this line can be stronger: 

> With each release, better changelogs are also coming, as we move towards adopting [conventional commits](https://www.conventionalcommits.org) for each PR that gets merged.

I'd change it to: 

1. Better changelogs are also coming as we move toward adopting conventional commits for each PR that gets merged.

Also note your `towards` should be `toward` (no 's') 

Sorry this looks pretty anal. Great post! 😄

Comment (adrianmcli) [reply src/blog/drizzle-150-a-new-beginning.md]: Thanks! Updated.
